packages/core/tools/_mixins.scss

```diff
@@ -38,6 +38,20 @@
 //
 
 @mixin visually-hidden() {
+  position: absolute;
+
+  // Absolute positioning has the unintended consequence of removing any
+  // whitespace surrounding visually hidden text from the accessibility tree.
+  // Insert a space character before and after visually hidden text to separate
+  // it from any visible text surrounding it.
```

Comment:
Does this mean that the examples / usage don't need any leading or trialing spaces in the markup?

Reply:
I guess it’t no longer technically needed, but probably still makes sense to leave it in as it makes the HTML more readable?

eg 

```html
<span class="nhsuk-u-visually-hidden">Countries beginning with</span>A
``` 

is not super clear vs 

```html
<span class="nhsuk-u-visually-hidden">Countries beginning with</span> A
```

?

Reply:
My loosely held thought is that the latter implies it's _necessary_, whereas it now seems like it isn't - but then perhaps something to address every time this class is used.

Reply:
It’s possibly still good practice to include the space anyway actually, for any modes where the CSS isn’t applied, like possibly some 'Reader' modes or print layouts?